frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/project/MacheteProjectResolver.java

```diff
@@ -33,4 +54,16 @@
   public boolean cancelTask(ExternalSystemTaskId taskId, ExternalSystemTaskNotificationListener listener) {
     return false;
   }
+
+  @SuppressWarnings({"guieffect:call.invalid.ui", "interning:not.interned"})
+  private Boolean hasSelectedGitMacheteTab(Project project) {
+    var toolWindowManager = ToolWindowManager.getInstance(project);
+    var toolWindow = toolWindowManager.getToolWindow(ToolWindowId.VCS);
+    if (toolWindow != null) {
+      var contentManager = toolWindow.getContentManager();
+      var gitMacheteContent = contentManager.findContent("Git Machete");
```

Comment:
Nit: do we keep tab name somewhere in properties?

Reply:
I can bundle it but note that the bundle cannot be used by plugin.xml
```
<extensions defaultExtensionNs="com.intellij">
        <changesViewContent tabName="Git Machete"
                            className="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteContentProvider"
                            predicateClassName="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteVisibilityPredicate"/>
...
```

Reply:
Aaa ok, leave as is then

Reply:
Yeah, I would rather leave it (we use `"Git Machete"` in two places only)